Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Application/Services/StringRandomizerService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MvvmSeed.Application.Data;
+using MvvmSeed.Domain.Services;
+using Realms;
+using System;
+using System.Linq;
+
+namespace MvvmSeed.Application.Services
+{
+    /// <summary>
+    /// Simple implementation of <see cref="IStringRandomizerService"/> that stores randomization info on a local database (Realm)
+    /// </summary>
+    public class StringRandomizerService : IStringRandomizerService
+    {
+        //TODO - When attempting to write UTs for this service, found out I couldn't because Realm class doesn't implement an interface/virtual methods. 
```

Comment:
Yep, I mentioned this on PR's description..

> One drawback is that Realm class doesn't implement an interface, so a framework other than Moq will be required for UnitTests (one capable of mockings non virtual methods, like TypeMock).

So, I won't merge it until I'm sure we can mock it with TypeMock or other framework.